src/congpairs.cc

```diff
@@ -31,6 +31,7 @@ using libsemigroups::Congruence;
 using libsemigroups::word_t;
 using libsemigroups::relation_t;
 using libsemigroups::RecVec;
+using libsemigroups::Partition;
```

Comment:
Is this supposed to be part of this commit? Just curious why it's been added (I'm obviously not an expert on this kind of stuff).